src/test/fuzz/secp256k1_ec_seckey_import_export_der.cpp

```diff
@@ -30,7 +30,7 @@ FUZZ_TARGET(secp256k1_ec_seckey_import_export_der)
         const bool exported = ec_seckey_export_der(secp256k1_context_sign, seckey.data(), &seckeylen, key32.data(), compressed);
```

Comment:
Not my area of expertise, but is `secp256k1_context_sign` even the correct name here, wasn't that deprecated in favor of [`SECP256K1_CONTEXT_NONE`](https://github.com/bitcoin-core/secp256k1/blob/master/include/secp256k1.h#L218)?